doc/usecases/polyculture/search_plants.md

```diff
@@ -0,0 +1,37 @@
+# Use Case: Search Plants
+
+## Summary
+
+- **Scope:** Plants Layer
+- **Level:** User Goal
+- **Actors:** App User
+- **Brief:** The user can search for plants to add to her map and sort results by different criterion.
+- **Status:** In Progress
+- **Assignee:** Moritz, Paul
+
+## Scenarios
+
+- **Precondition:**
+  - The user is logged in to the app.
+  - The plants layer is selected.
+- **Main success scenario:**
+  - The user types something into the search text box.  
+    This will search for partial matches of:
+    - names of specific plants (specie or variety)
+    - Binomial name (Latin name)
+    - German and English common names
+    - German and English synonyms
+    - Furthermore, other columns can be matched with extra syntax
+  - The results can be sorted by the following criteria:
+    - Companions of existing plants (default)
+    - Environmental fit
+    - Ecological value
```

Comment:
We think this is a bit better from usability (especially one button less to choose sort order from):

```suggestion
  - The results are ranked by:
    - Environmental fit
    - Ecological value
```

@Benjaminimal @Bushuo @badnames What do you think?
Did you already implement a sort order?
Is ranking more difficult to implement?

@Benjaminimal We think "Companions" are irrelevant when *searching* for plants. The grouping feature is much more useful in this case.